drivers/video/video_stm32_dcmi.c

```diff
@@ -338,6 +438,32 @@ static int video_stm32_dcmi_get_caps(const struct device *dev, struct video_caps
 	return video_get_caps(config->sensor_dev, caps);
 }
 
+static int video_stm32_dcmi_set_ctrl_snapshot(const struct device *dev, int value)
+{
+	struct video_stm32_dcmi_data *data = dev->data;
+
+	LOG_INF("Set Snapshot: %d\n", value);
+	if ((CONFIG_VIDEO_BUFFER_POOL_NUM_MAX == 1) && (value == 0)) {
+		LOG_DBG("Only one buffer so snapshot mode only");
```

Comment:
Worth a `LOG_ERR()`?

Reply:
I left it as debug as in many of the cases where one is using snapshot, they really only need/use 1 buffer,
although they can use multiple if they desire